src/migrations/20230502222224-add-user-device-id.ts

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  async up(db) {
+    try {
+      const result = db
+        .collection("users")
+        .createIndex({ device: 1 }, { unique: true, sparse: true })
+      console.log({ result }, "index created for users/device")
+    } catch (error) {
+      console.log({ result: error }, "error creating index for users/device")
```

Comment:
we should probably throw here

Reply:
i just copied how to other migrations were written

Reply:
ahah. it's not because I made dump migration that you have to copy them :)